ecs-cli/modules/cli/local/down_app.go

```diff
@@ -42,8 +42,30 @@ func Down(c *cli.Context) error {
 		network.Teardown(client)
 	}()
 
+	if err := optionsPreCheck(c); err != nil {
+		logrus.Fatalf("Tasks can be either created by local files or remote files")
+	}
+	// if c.Bool(flags.AllFlag) {
+	// 	return downLocalContainersWithFilters()
+	// }
+	// return downComposeLocalContainers()
+
+	if c.String(flags.TaskDefinitionFileFlag) != "" {
+		return downLocalContainersWithFilters(filters.NewArgs(
+			filters.Arg("label", taskDefinitionLabelValue+"="+c.String(flags.TaskDefinitionFileFlag)),
```

Comment:
nit: Instead of concatenating strings with the "+" operator, consider using [fmt.Sprintf](https://golang.org/pkg/fmt/#Sprintf). I find it more readable:
```suggestion
			filters.Arg("label", fmt.Sprintf("%s=%s, taskDefinitionLabelType, c.String(flags.TaskDefinitionFileFlag))
```

Reply:
ok thx. Fixed